readme.md

```diff
@@ -1,18 +1,18 @@
 # download-directory.github.io <img src="logo.svg" width="50" height="50" align="left">
 
-> Download just a sub directory from a GitHub repo
+> Download directory from a GitHub repo
 
 GitHub doesn’t let you download a single folder from a repo, which might be necessary when you just need a few files from a large repository.
 
 This tool will handle the download of all the files in a directory, in a single click, after you entered your token.
 
 The download starts automatically when you visit pass the link to the GitHub directory as `url` parameter, like:
 
-[**download-directory.github.io**`?url=https://github.com/mrdoob/three.js/tree/dev/build`](https://download-directory.github.io/?url=https://github.com/mrdoob/three.js/tree/dev/build)
+[**download-directory.github.io/**`?url=https://github.com/mrdoob/three.js/tree/dev/build`](https://download-directory.github.io/?url=https://github.com/mrdoob/three.js/tree/dev/build)
 
 You can also specify download filename by adding `filename` parameter, like:
 
-[**download-directory.github.io**`?url=https://github.com/mrdoob/three.js/tree/dev/build&filename=three-js-build`](https://download-directory.github.io/?url=https://github.com/mrdoob/three.js/tree/dev/build&filename=three-js-build) to save the file as **three-js-build.zip**.
+[**download-directory.github.io/**`?url=https://github.com/mrdoob/three.js/tree/dev/build&filename=three-js-build`](https://download-directory.github.io/?url=https://github.com/mrdoob/three.js/tree/dev/build&filename=three-js-build) to save the file as **three-js-build.zip**.
```

Comment:
All of these changes are unnecessary and should be reverted. The server is accessible without the slash and here it's just noise